packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -72,6 +72,31 @@ augmentFieldsDict(Posts, {
       },
     }
   },
+  totalDialogueResponseCount: {
+    resolveAs: {
+      type: 'Int', 
+      resolver: (post, _, context) => {
+        if (!post.debate) return 0;
```

Comment:
All of Ruby's code that includes use of the old `debate` field will need to be changed, but presumably he'll do that in his PR?